src/utils/options.ts

```diff
@@ -0,0 +1,46 @@
+import { OptionValidationResult } from "domain/entities/OptionValidationResult";
```

Comment:
Let's try to avoid a src/utils file. This is being used only from src/scripts/commands. I'd put the file there (in some folder?) and a filename as declarative as possible.  And the function/class can also call writeFileSync.